exampleCourse/questions/element/matching/question.html

```diff
@@ -0,0 +1,21 @@
+<pl-question-panel>
```

Comment:
The example course question for an element typically demonstrates several options for the element, from the most basic (only `answers-name` set) to each individual argument. The same matching can be used in all cases, the idea is to demonstrate how to use the element with multiple different options.

Also, can you add it to the demo homework?